experimental/GModule/GModule.jl

```diff
@@ -112,7 +112,12 @@ case that `S` is a number field and `R` is the ring of integers in `S`,
 for example `R = ZZ` and `S = QQ`)
```

Comment:
Those "examples" seem more like a TODO, perhaps you can also fill them in? (In a follow-up PR, though perhaps at least add "TODO" in this one so it won't be forgotten again.

The docstring above also claims 
> The syntax `M ⊗ phi` is supported.

which doesn't seem to be actually the case, so perhaps another TODO?